packages/react-native-sdk/src/components/LocalVideoView.tsx

```diff
@@ -1,5 +1,12 @@
-import React from 'react';
```

Comment:
This code is already in main and doesn't need review, GH is wrongfully showing it